integration-test/1955-chinese-parser.py

```diff
@@ -62,6 +68,130 @@ def test_san_francisco_osm(self):
+                u"name:tpi": u"Yunaitet Stet bilong Amerika",
+                u"name:tr": u"Amerika Birleşik Devletleri",
+                u"name:ts": u"United States",
+                u"name:tt": u"Америка Кушма Штатлары",
+                u"name:tw": u"USA",
+                u"name:ty": u"Fenua Marite",
+                u"name:tzl": u"Estats Viensiçeschti d'America",
+                u"name:udm": u"Америкалэн Огазеяськем Штатъёсыз",
+                u"name:ug": u"ئامېرىكا قوشما شتاتلىرى",
+                u"name:uk": u"Сполучені Штати Америки",
+                u"name:ur": u"ریاستہائے متحدہ امریکہ",
+                u"name:uz": u"Amerika Qoʻshma Shtatlari",
+                u"name:vec": u"Stati Unìi de la Mèrica",
+                u"name:vep": u"Amerikan Ühtenzoittud Valdkundad",
+                u"name:vi": u"Hoa Kỳ",
+                u"name:vls": u"Verênigde Stoaten van Amerika",
+                u"name:vo": u"Lamerikän",
+                u"name:wa": u"Estats Unis d' Amerike",
+                u"name:war": u"Estados Unidos",
+                u"name:wo": u"Diwaan-yu-Bennoo yu Aamerig",
+                u"name:wuu": u"美利坚合众国",
+                u"name:xal": u"Америкин Ниицәтә Орн Нутгуд",
+                u"name:xh": u"IYunayithedi Steyitsi",
+                u"name:xmf": u"ამერიკაშ აკოართაფილი შტატეფი",
+                u"name:yi": u"פאראייניקטע שטאטן פון אמעריקע",
+                u"name:yo": u"Orílẹ́ède Orilẹede Amerika",
+                u"name:yue": u"即美利堅合眾國",
+                u"name:za": u"Meijgoz",
+                u"name:zea": u"Vereênigde Staeten",
+                u"name:zh": u"美國",
+                u"name:zh_pinyin": u"Měiguó",
+                u"name:zu": u"i-United States",
+                u"not:official_name:vi": u"Hợp chủng quốc Hoa Kỳ",
+                u"official_name": u"United States of America",
+                u"official_name:en": u"United States of America",
+                u"official_name:eo": u"Unuiĝintaj Ŝtatoj de Ameriko",
+                u"official_name:fr": u"États-Unis d'Amérique",
+                u"official_name:pl": u"Stany Zjednoczone Ameryki",
+                u"official_name:pt": u"Estados Unidos da América",
+                u"official_name:sv": u"Amerikas förenta stater",
+                u"official_name:vi": u"Hợp chúng quốc Hoa Kỳ",
+                u"official_name:vo": u"Tats-Pebalöl Nolüda-Meropa",
+                u"old_name:vi": u"Mỹ Lợi Kiên;Ma Ly Căn;Nhã Di Lý",
+                u"place": u"country",
+                u"population": u"324720797",
+                u"short_name": u"USA",
+                u"short_name:en": u"USA",
+                u"short_name:es": u"EE.UU.",
+                u"short_name:vi": u"Mỹ",
+                u"source": u"openstreetmap.org",
+                u"sqkm": u"9826675",
+                u"wikidata": u"Q30",
+                u"wikipedia": u"en:United States"
+            }))
+
+        self.assert_has_feature(
+            16, 10482, 25330, 'places',
```

Comment:
Since the rendering logic changes per zoom and by kind... please add to test
- as a low zoom tile (`4/3/6`) (change test to test `4, 3, 6` not the zoom 16 coord)
- for `kind: country` (add kind to the output test):

![image](https://user-images.githubusercontent.com/853051/127192047-14989d38-11b6-45ef-a063-e2ab5d3301a5.png)

Reply:
> Since the rendering logic changes per zoom and by kind... please add to test
> 
> * as a low zoom tile (`4/3/6`) (change test to test `4, 3, 6` not the zoom 16 coord)
> * for `kind: country` (add kind to the output test):
> 
> ![image](https://user-images.githubusercontent.com/853051/127192047-14989d38-11b6-45ef-a063-e2ab5d3301a5.png)

I was able to find an example of how to do a low zoom integration test, so just copied some code, please see in the newest commit `test_united_states`